website/siteConfig.js

```diff
@@ -51,7 +51,6 @@ const siteConfig = {
   // For no header links in the top nav bar -> headerLinks: [],
   headerLinks: [
     {doc: 'whatisexo', label: 'Docs'},
-    {doc: 'APILanding', label: 'API'},
```

Comment:
I could change this to `SDK`. And under that:

- Introduction
- Web APIs (currently Javascript "tech stack")
- Naive Integrations (currently native "tech stack")
- Future integrations (the idea list)